src/themes/default/components/core/AddToCart.vue

```diff
@@ -19,7 +19,7 @@ export default {
       this.$forceUpdate()
     },
     canBeAdded (product) {
-      return formatProductMessages(product.errors) !== ''
+      return formatProductMessages(product.errors) !== '' || product.qty <= 0
```

Comment:
```suggestion
      return formatProductMessages(product.errors) !== ''
```
This check is redundant since we have vuelidate

Reply:
this is to block the button